force-app/main/default/lwc/serviceDeliveryRow/serviceDeliveryRow.js

```diff
@@ -398,7 +543,7 @@ export default class ServiceDeliveryRow extends LightningElement {
                 }
             });
 
-            if (hasContact) {
+            if (this.hasContactField && contactId !== "") {
```

Comment:
I think we can just check if (contactId) here since it should be undefined if the field is not in the field set, I'm not sure it would ever be an empty string but it might be undefined if the value passed is null or the localFieldSet doesn't have the field.

Reply:
@bmuniswamy - previously when we were defaulting the contact and programEngagement from CaseMan we ran handleGetServicesEngagements but now that the contact field is no longer required we run into an issue where the program engagement is not triggering the filter on the service.
![Screen Shot 2020-06-18 at 2 07 21 PM](https://user-images.githubusercontent.com/450473/85062111-8a6a5f00-b16d-11ea-8372-6f475ae5f3a2.png)
To test this we can set the default values values in the UI.js class like so... ```
@api defaultValues = {
        ProgramEngagement__c: "a011100000iNmpIAAS",
        Contact__c: "0031100001fGglpAAC",
    };
```